src/go/k8s/pkg/resources/configmap.go

```diff
@@ -164,6 +167,16 @@ func (r *ConfigMapResource) createConfiguration(
 	}
 
 	cr.AdminApi[0].Port = clusterCRPortOrRPKDefault(c.AdminAPI.Port, cr.AdminApi[0].Port)
+	cr.AdminApi[0].Name = internal
+	externalAdminAPI := config.NamedSocketAddress{
```

Comment:
so there's always going to be internal and external even if external connectivity is not enabled?

Reply:
addressed.